### PR TITLE
add merge script

### PR DESCRIPTION
**Pull request by felixcheung**
Ported the merge script from spark main repo

### CODE REVIEW

File: doap.rdf [hunk 1]
```diff
@@ -35,6 +35,9 @@
     <programming-language>Java</programming-language>
     <programming-language>Scala</programming-language>
     <programming-language>Python</programming-language>
+    <programming-language>R</programming-language>
```
Comment (srowen) [inline doap.rdf]: This is not directly related right? but fine

**Review by srowen — APPROVED** [review]
Although duplication is a bit undesirable there's no easy way to share it otherwise. Seems fine, especially if you were able to trim out parts of the script that don't apply.

File: doap.rdf (hunk 1 above)
Comment (HyukjinKwon) [reply doap.rdf]: Hmmm .. I'd leave this change out though. Seems we need to fix 31L too.

**Review by HyukjinKwon — APPROVED** [review]

File: doap.rdf (hunk 1 above)
Comment (felixcheung) [reply doap.rdf]: haha ok. actually I'm not sure R is supported as a value, couldn't open some broken links

```
Not Found
The requested URL /languages.html was not found on this server.
```
https://projects.apache.org/languages.html
